src/Symfony/Bundle/SecurityBundle/Security/FirewallContext.php

```diff
@@ -25,7 +25,7 @@ class FirewallContext
     private $exceptionListener;
     private $config;
 
-    public function __construct(array $listeners, ExceptionListener $exceptionListener = null, FirewallConfig $config = null)
+    public function __construct(/* iterable */ $listeners, ExceptionListener $exceptionListener = null, FirewallConfig $config = null)
```

Comment:
OK, I missed that. Then we should be clear about that in the CHANGELOG at least.